src/pykiso/cli.py

```diff
@@ -90,12 +90,41 @@ def check_file_extension(
     return paths
 
 
+class CommandWithOptionalFlagValues(click.Command):
+    def parse_args(self, ctx, args):
+        """Translate any flag `--junit=value` as flag `--junit` with changed flag_value=value"""
+        # filter flags
+        flags = [
+            flag
+            for flag in self.params
+            if isinstance(flag, click.Option)
+            and flag.is_flag
+            and not isinstance(flag.flag_value, bool)
+        ]
+        junit_prefixes = {
+            flag_str: flag
+            for flag in flags
+            for flag_str in flag.opts
+            if flag_str.startswith("--junit")
+        }
+
+        for index, arg in enumerate(args):
+            arg = arg.split("=")
+            if len(arg) < 2:
+                continue
+            junit_prefixes[arg[0]].flag_value = arg[1]
+            args[index] = arg[0]
+        result_args = super(CommandWithOptionalFlagValues, self).parse_args(ctx, args)
+        return result_args
```

Comment:
To make this generic:
```suggestion
class CommandWithOptionalFlagValues(click.Command):
    def parse_args(self, ctx, args):
        """Translate any flag `--junit=value` as flag `--junit` with changed flag_value=value"""
        # filter flags
        flags = [
            flag
            for flag in self.params
            if isinstance(flag, click.Option)
            and flag.is_flag
            and not isinstance(flag.flag_value, bool)
        ]
        junit_prefixes = {
            flag_str: flag
            for flag in flags
            for flag_str in flag.opts
            if flag_str.startswith("--junit")
        }

        for arg_index, arg in enumerate(args):
            arg = arg.split("=")
            if len(arg) != 2:
                continue
            arg_name, arg_value = arg
            for flag in flags:
                if arg_name not in flag.opts:
                    continue
                flag.flag_value = arg_value 
                args[arg_index] = arg_name
                
        result_args = super(CommandWithOptionalFlagValues, self).parse_args(ctx, args)
        return result_args
```